packages/shared/src/components/checklist/SquadChecklistCard.tsx

```diff
@@ -5,19 +5,18 @@ import { Squad } from '../../graphql/sources';
 import InteractivePopup, {
   InteractivePopupPosition,
 } from '../tooltips/InteractivePopup';
-import useSidebarRendered from '../../hooks/useSidebarRendered';
-import { Modal } from '../modals/common/Modal';
-import { ModalKind } from '../modals/common/types';
 import { useChecklist } from '../../hooks/useChecklist';
 import AnalyticsContext from '../../contexts/AnalyticsContext';
 import { AnalyticsEvent, TargetType } from '../../lib/analytics';
+import { useViewSize, ViewSize } from '../../hooks';
+import { Drawer } from '../drawers';
 
 const SquadChecklistCard = ({ squad }: { squad: Squad }): ReactElement => {
   const { steps, isChecklistVisible, setChecklistVisible, isChecklistReady } =
     useSquadChecklist({
       squad,
     });
-  const { sidebarRendered } = useSidebarRendered();
+  const isMobile = useViewSize(ViewSize.MobileL);
```

Comment:
Instead of relying on javascript, can we render both and rely on CSS media queries and hide the element we don't want to see?

Reply:
The only potential drawback to that is if we trigger any events or things like that, if both are rendered, we would be duplicating them? maybe something to consider on a case by case basis

Reply:
Yes, one thing that easily comes to mind is closing on outside click. Either one of them will think you are always clicking outside its elements, hence closing it. There's probably some workaround, but let's see what are the pros and cons on both ends.

Reply:
Good point Milan.
The idea is that we should use CSS media queries wherever possible as the default.

There some edge cases where we need it in JS evals, then we prefer the useViewSize hook.